src/modules/Dropdown/Dropdown.js

```diff
@@ -425,64 +422,57 @@ export default class Dropdown extends Component {
         if (openOnFocus) this.open()
       }
       if (!this.state.open) {
-        document.addEventListener('keydown', this.openOnArrow)
-        document.addEventListener('keydown', this.openOnSpace)
-      } else {
-        document.addEventListener('keydown', this.moveSelectionOnKeyDown)
-        document.addEventListener('keydown', this.selectItemOnEnter)
+        eventPool.sub('keydown', [this.openOnArrow, this.openOnSpace])
       }
-      document.addEventListener('keydown', this.removeItemOnBackspace)
```

Comment:
@levithomason Thanks for clarification, I've restored event handlers and added a fix. Now `eventPull` [handles only](https://github.com/Semantic-Org/Semantic-UI-React/pull/1733/commits/ffa4a434373c57efc59fdc3f27d03a4b21ede4bb) unique events